### ISSUE TITLE
Support for relative paths

### ISSUE THREAD

**Issue by DaelDe** [marcjulianschwarz/obsidian-file-link#5]
I store the vault in subversion and want to link files in the same repository. To make this work on all PCs where the repository is fetched, I have to insert relative links to files external to the vault. 

I tried to modify the path after inserting, but it does not seem to work.

**Comment by tsbertalan** [comment]
Of course, anything is possible with the right scripting, the question is whether you want to override the default behavior for `file://` links that you somehow decide are relative, or just keep farming the actual behavior out to the markdown parser.

I do find that, following [this answer](https://stackoverflow.com/questions/4443597/node-js-execute-system-command-synchronously), I'm able to pop up a Windows Explorer window to an arbitrary path using the Obsidian console like this:

```javascript
const {execSync} = require('child_process');
const result = execSync('explorer.exe C:\Users\tom\Dropbox');
```

This throws a checkExecSyncError in the console, but the Explorer window does open.

That's Windows-specific, but I imagine you could do some platform checks to have a Windows/Linux/Darwin case block.
